Frontend/ui-customer/src/Screens/CustomerUtils/MyProfileScreen/Controller.purs

```diff
@@ -151,6 +154,10 @@ eval (EmailIDEditTextAction (PrimaryEditText.TextChanged id value)) state = do
 
 eval (UpdateButtonAction (PrimaryButton.OnClick)) state = do
   _ <- pure $ hideKeyboardOnNavigation true
+  if state.data.gender /= state.data.editedGender then do
+      _ <- pure $ firebaseLogEvent if state.props.fromHomeScreen then "banner_gender_celected" else "profile_gender_selected"
```

Comment:
@aryanguptaSG please correct the spelling "banner_gender_celected" => "banner_gender_selected"